stackslib/src/chainstate/nakamoto/signer_set.rs

```diff
@@ -244,7 +271,7 @@ impl NakamotoSigners {
                                 "signer".into(),
                                 Value::Principal(PrincipalData::from(signing_address)),
                             ),
-                            ("num-slots".into(), Value::UInt(signer.slots.into())),
+                            ("weight".into(), Value::UInt(signer.slots.into())),
```

Comment:
I am confused about the naming here.
`signer.slots` are rewards slots (=weight), not signer slots (=num-slots). Correct?

Reply:
Yes, that's pretty much correct.

signer.slots are (roughly) the reward slots associated with that signer. This is also their "weight" in the aggregate public key, and in the DKG voting.

num-slots is the number of stackerdb slots that the signer needs for communication -- this is a constant for every signer.